src/Calendar.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import { useQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+
+import EventDialog from './components/EventDialog';
+import NewEventDialog from './components/NewEventDialog';
+
+import './main.scss';
+
+const query = gql`
+  query GetEvents {
+    getEvents {
+      id
+      title
+      start
+      end
+      description
+    }
+  }
+`;
+const Calendar = () => {
+  const [openEvent, setOpenEvent] = useState(false);
+  const [openNewEvent, setOpenNewEvent] = useState(false);
+  const [selectedEvent, setSelectedEvent] = useState();
+  const [selectedDate, setSelectedDate] = useState();
+
+  const handleClickOpen = selectedEvent => {
+    setSelectedEvent(selectedEvent);
+    setOpenEvent(true);
+  };
+
+  const handleClose = () => {
+    setOpenEvent(false);
+  };
+
+  const handleClickOpenNewEvent = selectedDate => {
+    setSelectedDate(selectedDate);
+    setOpenNewEvent(true);
+  };
+
+  const handleCloseNewEvent = () => {
+    setOpenNewEvent(false);
+  };
+
+  const { data, loading, error } = useQuery(query, { pollInterval: 1000 });
```

Comment:
Is there a way in GraphQL to signal a change before `pollInterval` happens? Currently when adding an event or removing there's a delay between it appearing / disappearing.

Reply:
Cool!